netlify.toml

```diff
@@ -10,7 +10,4 @@ HUGO_VERSION = "0.88.1"
 HUGO_THEME = "repo"
 
 [context.production]
-command = "npm run build:production"
-
-[context.production.environment]
-HUGO_ENV = "production"
+command = "npm install && npm run build:production"
```

Comment:
@LisaFC: this is the main change that fixes the production build. My bad for not propagating the preview fix to this line too.